lib/elixir/lib/enum.ex

```diff
@@ -1653,6 +1653,9 @@ defmodule Enum do
   It assumes that the sample being returned can fit into memory;
   the input `enumerable` doesn't have to, as it is traversed just once.
 
+  If a range is passed into the function it will not be expanded so it
```

Comment:
I think it doesn't hurt to mention complexity, we already do this in docs for other functions.